src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,26 @@
+const State          = require('../../_common/storage').State;
+const Client         = require('../../app/base/client');
+const BinarySocket   = require('../../app/base/socket');
+const getElementById = require('../../../javascript/_common/common_functions').getElementById;
+
+const ResponsibleTrading = (() => {
+
+    const onLoad = () => new Promise(resolve => {
```

Comment:
I don't think our onload functions are promises, are they? in binary_loader.js it's just calling `active_script.onLoad();`

Reply:
corrected